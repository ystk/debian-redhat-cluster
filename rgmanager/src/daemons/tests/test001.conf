<?xml version="1.0"?>
<!-- inline resource group -->
<cluster>
<rm>
	<service name="test1"/>
</rm>
</cluster>
