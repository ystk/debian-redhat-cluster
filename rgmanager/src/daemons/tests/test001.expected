=== Resources List ===
Resource type: service [INLINE]
Instances: 1/1
Agent: service.sh
Attributes:
  name = test1 [ primary unique required ]

=== Resource Tree ===
service {
  name = "test1";
}
